uw-frame-components/portal/widgets/partials/widget-card.html

```diff
@@ -23,7 +23,7 @@
     </div>
 
     <md-card-header-text>
-      <span class="md-title" style="text-align: center;" aria-label="{{ widget.title }}" tabindex="0">
+      <span class="md-title" style="text-align: center;" aria-label="{{ widget.title }}" aria-hidden="true">
```

Comment:
Is it weird to `aria-label` an element that's also `aria-hidden`?
Does this mean that if the widget is in `MAINTENANCE` lifecycle state then screen readers won't read the title of the widget at all? 

I think this change is happening to make the read-first description more sensible in the context of the widget. But probably usually users don't want to read the description at all?

Consider instead aria-hiding the info md-button, not hiding this md-title span, and adding a link to the details page, visible only to screen readers, with the app description included as the description of what that link is going to take you to? Disclaimer: haven't tested this hypothetical alternative, so maybe its accessibility is bad.